books/tour/src/borrowings.md

```diff
@@ -1,25 +1,19 @@
-# Borrowings [ZA]
+# Borrowings [XA]
 
 Borrowings allow importing words from other languages which would be difficult
-to translate and express using root words, such as proper names, names of species and cultural concepts.
+to translate and express using root words, such as proper names, names of
+species and cultural concepts.
 
-A borrowing starts with the particle **za**, followed by the borrowed word, then
+A borrowing starts with the particle **xa**, followed by the borrowed word, then
 a mandatory pause. The borrowed word should be a phonetic retranscription in
 the phonology and morphology of eberban. Incompatible chains of vowels, or
 pauses between a term made of multiple foreign words, should be borrowed using
 *h*. Incompatible chains of consonants should be borrowed using a buffer vowel,
-such as *y*. A word starting with *l/m/r* must be prefixed with another letter,
-as they are forbidden at the begining of every word.
```

Comment:
Regarding this:
> A word starting with *l/m/r* must be prefixed with another letter,
such as *q*.
as they are forbidden at the begining of every word.

Can we really remove this restriction?

Reply:
Prefixing a `'` before a sonorant initial word should be enough :p